Geometry/HcalCommonData/src/CaloSimParametersFromDD.cc

```diff
@@ -8,7 +8,7 @@
 #include <iostream>
 #include <iomanip>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
@bsunanda , likely this should be commented out

Reply:
Taken care of